### DISCUSSION TITLE
Cloud Firestore em produção ?

### DISCUSSION THREAD

**Discussion by romulosardinha** [frontendbr/forum#1302] [Perguntas e Respostas]
Alguém usando Cloud Firestore em produção ?
Como eles estão em beta ainda estou com medo de colocar meus novos clientes nele e futuramente ter mudanças que me forçariam a reescrever o código. 

Já estou usando o Firebase Realtime em produção tranquilamente a um tempo, mas como o Firestore vai se tornar o padrão do Google, estou nesta dúvida.

**Comment by luizbills** [comment]
Eu não recomendo nunca usar em produção tecnologias muito novas ou em beta. Podem ser descontinuadas, podem mudar, podem ter vulnerabilidades, bugs, etc

Esse Cloud Firestore seria um Firebase com os problemas resolvidos?

**Comment by romulosardinha** [comment]
Cara, eu não vejo problemas em nenhum produto que eu esteja usando firebase realtime, mas entendi o que tu disse: não, o Firestore é um um nosql orientado a documentos, então é uma forma diferente do realtime. Porém ainda está em beta, segundo as publicações oficiais do google ainda sairão novas features para ele.
